aten/src/ATen/native/cuda/LinearAlgebra.cu

```diff
@@ -1,3 +1,4 @@
+#define TORCH_ASSERT_NO_OPERATORS
```

Comment:
Can we remove `#include <ATen/native/BatchLinearAlgebra.h>?`
https://github.com/pytorch/pytorch/pull/67833#discussion_r777410369

Reply:
No, `unpack_pivots_stub` is defined in that header. However, the header now compiles fine under `TORCH_ASSERT_NO_OPERATORS` because `Tensor` is forward declared instead of being included.